### PR TITLE
Product listing

### PR DESCRIPTION
**Pull request by Rockyraven**
refactor of filter box and wishlist icon

### CODE REVIEW

File: src/component/card/ProductCard.jsx [hunk 1]
```diff
@@ -1,28 +1,34 @@
-import React from 'react'
+import React, { useState } from 'react'
 import './productcard.css'
 export const ProductCard = ({ title, brand, discount, price, actualPrice, imgSrc, rating }) => {
+    const [ BgColor , SetBgColor ] = useState("");
```
Comment (arjun-computer-geek) [inline src/component/card/ProductCard.jsx]: don't give any variable name with capital letter, not event function

**Review by arjun-computer-geek — APPROVED** [review]
code is nicely formatted. Good and cleaner code.

File: src/pages/product/Product.js [hunk 2]
```diff
@@ -1,23 +1,35 @@
...
+    return data
+
+  }
+
+  const filteredData = filterItem(product, "Formal")
+  console.log(filteredData)
   return (
     <>
+  {/* <button onClick={filteredData.map(item => <ProductCard key={item._id} title={item.title} brand={item.brand} rating={item.rating} price={item.price} discount={item.discount} actualPrice={item.actualPrice} imgSrc={item.image} />)}>click</button> */}
+      <div className="map">
```
Comment (Shubhamkr2610) [inline src/pages/product/Product.js]: remove unused code

**Review by Shubhamkr2610 — APPROVED** [review]
Appreciation: your code is clean and readable, easy to understand . your logic is good . You are good to go

File: src/component/card/ProductCard.jsx [hunk 3]
```diff
@@ -1,28 +1,36 @@
...
-                    <div class="price-content">
-                        <p class="price">₹{price} </p>
-                        <p class="actual-price"> {actualPrice}</p>
-                        <p class="discount">{discount}</p>
+                    <div className="price-content">
+                        <p className="price">₹{price} </p>
+                        <p className="actual-price"> {actualPrice}</p>
+                        <p className="discount">{discount}</p>
                     </div>
                     <button className='buy'>ADD TO CARD</button>
```
Comment (anshtripathi01) [inline src/component/card/ProductCard.jsx:14]: There's spelling mistake in "Add to Card ". Replace it with "Add to Cart".

File: src/index.css [hunk 4]
```diff
@@ -5,9 +5,10 @@
     --primary-color: rgb(173, 171, 171);
     --default-color: white;
     --loginbackground-color: rgb(219, 219, 219);
-    --signbackground-color: rgb(223, 219, 219);
+    --signbackground-color: rgb(243, 243, 243);
```
Comment (anshtripathi01) [inline src/index.css:8]: Use hex-code for colors

**Review by anshtripathi01 — APPROVED** [review]
make above changes else everything is good. You can merge it

File: src/pages/product/Product.js (hunk 2 above)
Comment (Rockyraven) [reply src/pages/product/Product.js]: done

**Pull request merged**